Changes

```diff
@@ -36,6 +36,10 @@ Next version (4.05.0):
    and Damien Doligez, discussion with Alain Frisch and Xavier Leroy,
    feature request from the Coq team)
 
+- PR#7172, GPR#970: add extra (ocamlc -config) options
+  int_size, word_size, ext_exe
+  (Gabriel Scherer, request by Daniel Buenzli)
```

Comment:
I actually like umlaüten, so Bünzli is the spelling I tend to use, no idea why this one was different. There are also two clearly wrong "Bunzli" in the Changes file (I think it is Alain's fault), so we could do a pass to harmonize this. I'll have a look at whether other names should be fixed as well.